### DISCUSSION TITLE
Adding index to repeater rows

### DISCUSSION THREAD

**Discussion by ecoy1** [awcodes/filament-table-repeater#127] [Q&A] [answered]
Hi,

Is there a way of adding an index for each of the repeaters? I saw a code snippet that works on the original Repeater using ->itemLabel however that doesnt seem to have an impact on TableRepeater. Let me know!

All the best!
Ece

**Comment by awcodes** [comment]
are you referring to row numbers?

**Reply by ecoy1** [reply]
Yes

**Reply by awcodes** [reply, marked as answer]
Not available at the moment. I'll make a note though to look at adding something for it.
